push-agents/apns-agent/src/main/java/org/aerogear/push/apns/ios/PushyApnsSender.java

```diff
@@ -80,46 +84,43 @@ public void sendPushMessage(final Variant variant, final Collection<String> toke
             }
         }
 
-        final ApnsClient apnsClient = buildApnsClient(iOSVariant);
-
-        final Future<Void> connectFuture = apnsClient.connect(ApnsClient.DEVELOPMENT_APNS_HOST, ApnsClient.DEFAULT_APNS_PORT);
-        connectFuture.addListener(future -> {
-
-
-            if (future.isSuccess()) {
-                logger.debug("connected!");
-
+        final ApnsClient apnsClient;
+        {
```

Comment:
Why this block?

Reply:
for `final` - the push team does it that way, and last year we adopted the same, when introducing push